src/solc_0.8/polygon/child/land/PolygonLandBaseToken.sol

```diff
@@ -158,7 +158,7 @@ contract PolygonLandBaseToken is ERC721BaseToken {
             address owner = _ownerOf(id1x1);
             require(owner != address(0), "token does not exist");
             require(owner == from, "not owner in _transferQuad");
-            _owners[id1x1] = uint256(sha256(abi.encodePacked(to)));
+            _owners[id1x1] = uint256(uint160(address(to)));
```

Comment:
Does `uint256(uint160(address(to)));` give the same output as `uint256(sha256(abi.encodePacked(to)));` ?

Reply:
No, the original solution for converting "address" to "uint256" was wrong